Updated logo, hamburger icon, margins header and main.

Martine04384 opened the pull request:
### What does this PR do?
Updated logo and hamburger icon on sm and md screens
Updated margins in main between header and breadcrumbs and main/button and footer om sm and md screens.

### Description of Task to be completed?
Compare and test. 

### How should this be manually tested?
Check in the browser and compare to Figma. 

### Any background context you want to provide?
A detected discrepancy between Figma and the file was detected after testing; an update was needed.

Martine04384 committed:
Updated logo, hamburger icon, margins header and main.

sulenchy committed:
fix merge conflict in edit profile admin page

inastefansdottir reviewed (changes requested):
This looks fantastic, Martine! It's getting really close to matching the Figma file, great job!

I noticed that some links, like the logo, aren’t showing up due to the file structure changes. You can fix this by adding ../ to the logo path.

Also, I really like the hamburger menu you're using, but we need to follow the Figma design with pixel perfection. It would be great if you could update it to match the design.

That's all, everything else looks great! Keep up the awesome work!

Picture for reference, current changes on the branch is on the left and the figma design is on the right:
![Screenshot 2025-03-28 003908](https://github.com/user-attachments/assets/10ff7e29-5b46-4072-b755-158ec337b9c3) ![image](https://github.com/user-attachments/assets/9fa71f95-fe48-4c7f-a177-ad4a7ff27b3b)

Martine04384 committed:
Updated footer img sm screens and filepath logo.

Martine04384 commented:
Hi, thanks for the feedback!

I tried to fix the footer img on sm screens. Now its the same size as in figma nut a bit zoomed in. I kind of have to choose between the right look of the circles (as above) or the same size (as in this new code). The margins is updated so it looks better but not perfect. 
 
 I will have a look at the nav!
 
<img width="992" alt="image" src="https://github.com/user-attachments/assets/1a37ce9a-3080-4fd5-92b7-f84234351b83" />

Martine04384 committed:
Updated hamburger icon.

Martine04384 commented:
Hi, seems like one of the hamburger icon "lines" got lost the last time. I've added the line and adjusted the first line to make it look more like in Figma. 
<img width="334" alt="image" src="https://github.com/user-attachments/assets/2af1c443-487b-47c9-a323-d8211541f3ce" />
<img width="397" alt="image" src="https://github.com/user-attachments/assets/cab4ccd2-ec2b-4a2f-b7a1-bcc6b7362c7d" />

The first img is how it looks like now and the second is the Figma design.

inastefansdottir commented:
Hi Martine its looking really great, I appreciate you making the changes that I requested! The two circles at the bottom look a lot better and more like the figma file. The hamburger menu looks pretty similar to the figma file with the extra line added, looking at it side by side. There are only tiny things that I noticed that could be improved for it to be perfect! The logo on the mobile version has too much space around it compared to the figma file, if you could lessen the space, that would be perfect. Left pic is live server and right is the figma file:

![Screenshot 2025-03-31 183827](https://github.com/user-attachments/assets/338d707b-c30d-4c01-a1c5-fd2e1145dea7) ![image](https://github.com/user-attachments/assets/ed1bf5c4-c8b7-4125-8419-d418c7cac5e9)

Also the logo on the mobile version is  a bit too small compared to the figma file, could you make it a bit bigger to fit the figma file better? Left pic is live server and right is the figma file:

![image](https://github.com/user-attachments/assets/c72cdc3a-49eb-4ceb-a878-d9632aaaa93b) ![Screenshot 2025-03-31 183752](https://github.com/user-attachments/assets/9aa8e2b1-5b01-46af-943e-b40a57fc8429) 

Its looking so close to perfection!

Martine04384 committed:
Updated logo header, small adjustments hamburger menue, updated margins around breadcrumbs according to figma.

Martine04384 commented:
Hi, thanks for feedback! I have some problems with the logo header file. It can only be shown in 64px x 64px or 176px x 176px INCLUDED padding. I have tried everything (containers, scaling, transforming, positioning within a container), and I can't remove the padding with Tailwind. Therefore, I have exported the image file from Figma and added it to the image in our code. But I'm not sure about the quality of the image.

I also noticed a few lines off in the hamburger menu and fixed that. 
And noticed the breadcrumbs where mt and mb 12 on lg screens and not 20 as in figma. But my tailwind stopped working again, so let me know if it worked! :)

Martine04384 committed:
Final update breadcrumbs.

inastefansdottir commented:
Hi Martine, The mobile version looks fantastic! It’s really close to the Figma file, and I don’t think any more changes are needed there. Great work!

Sorry for being so picky, but on the desktop version, the logo still feels a bit small compared to the Figma file, the padding looks fine on the logo, the size just needs to be increased a bit. Also the spacing around the breadcrumbs seems a bit too much. I know you worked on that, but since Tailwind stopped working, that might have caused some issues. If it happens again, you can try running npm run build to fix it. It is really odd that the mobile version has different inputs and text like the "positions" input and the "DOB" text than the desktop version, but I think it was a good idea to follow the mobile version.

![image](https://github.com/user-attachments/assets/a1eb872f-05e4-4a42-a3b9-016427881df1) ![image](https://github.com/user-attachments/assets/1b54e807-ae01-4709-ae09-ba526ea14f0c)

I also noticed that the margins inside the box needs to be increased according to the figma file, but I don't want to put too much work on you so I asked Adrian to make a bug issue for me to fix it.

Thanks again for your hard work!

inastefansdottir commented:
Hi again, I noticed that there is already a png img of the hamburger menu in the repo, could you use that one instead to keep consistency?

![image](https://github.com/user-attachments/assets/beceb185-4380-4f8c-ac81-f8fefb564de4)

Martine04384 committed:
Updated logo size desktop, hamburger icon, padding breadcrumbs, size main "box" and padding inside main, padding top h1, padding bottom button.

Martine04384 commented:
Hi, okey now I am finally back with working Tailwind! Makes things a lot easier :D 

I have fixed:
- Logo desktop should be as big as in Figma. 
- Did a little mixup with my other branch so I have fixed the breadcrumbs again. 
- Switch the hamburger icon to img. 
- Fixed the size of the box in main and the padding inside this box. 
- I also added a bit more space over the H1 and under the button. 

Hope this looks better!

inastefansdottir committed:
Merge branch 'develop' into 416-Fix-margins-and-logo-sm-screens

inastefansdottir reviewed (approved):
This looks perfect, I appreciate you making all the changes I suggested. I am satisfied with the changes and happy to approve and merge the branch. Great work!
